calico-enterprise_versioned_docs/version-3.18-2/variables.js

```diff
@@ -1,12 +1,12 @@
 const releases = require('./releases.json');
 
 const variables = {
-  releaseTitle: 'v3.18.0-2.0',
+  releaseTitle: 'v3.18.1',
   prodname: 'Calico Enterprise',
   prodnamedash: 'calico-enterprise',
   version: 'v3.18',
-  baseUrl: '/calico-enterprise/3.18',
-  filesUrl: 'https://downloads.tigera.io/ee/v3.18.0-2.0',
+  baseUrl: '/calico-enterprise/3.18-2',
```

Comment:
```suggestion
  baseUrl: '/calico-enterprise/3.18',
```

Reply:
Fixed.

Reply:
@ctauchen baseUrl: '/calico-enterprise/3.18' did not work. Had to change it to 3.18.1 for manifests to show up correctly. Also, manifest URLs are all fine except helm...it is showing 3.18.2. I'll check with Dan to see if something is not changed yet on Eng side. Any ideas?